config/initializers/simple_form_bootstrap.rb

```diff
@@ -0,0 +1,148 @@
+SimpleForm.setup do |config|
```

Comment:
> I read somewhere that bootstrap isn't the greatest for accessibility

Yeah, I think that is more about the [Components](http://getbootstrap.com/components/) (which I don't think we're using...?) than the [CSS](http://getbootstrap.com/css/), but I don't know too many details. Googling brought up

* http://getbootstrap.com/getting-started/#accessibility
* https://paypal.github.io/bootstrap-accessibility-plugin/

> I am not sure what using simple_form_bootstrap is doing exactly here

Yeah, I find the SimpleForm element config syntax very confusing :confused: but this is what it generated out of the box.

> not sure if that will be a problem

We were already using Bootstrap, and the generated HTML is very similar to what we had before (though my investigation wasn't exhaustive). At the very least there shouldn't be any regressions in terms of accessibility, besides https://www.pivotaltracker.com/story/show/105961466.